megazords/ios-rust/MozillaTestServices/MozillaTestServices.xcodeproj/project.pbxproj

```diff
@@ -70,13 +70,15 @@
 		39F5D7642956161E004E2384 /* Operation+.swift in Sources */ = {isa = PBXBuildFile; fileRef = 39F5D7632956161E004E2384 /* Operation+.swift */; };
 		39F5D766295616E3004E2384 /* NimbusBuilder.swift in Sources */ = {isa = PBXBuildFile; fileRef = 39F5D765295616E3004E2384 /* NimbusBuilder.swift */; };
 		45CC574A28AD9C86006D55AA /* errorsupport.udl in Sources */ = {isa = PBXBuildFile; fileRef = 45CC574828AD9C31006D55AA /* errorsupport.udl */; };
+		AB65933C2A0C527900DBF059 /* remote_settings.udl in Sources */ = {isa = PBXBuildFile; fileRef = AB65933B2A0C524E00DBF059 /* remote_settings.udl */; };
+		AB6593402A0C52A000DBF059 /* remote_settings.swift in Sources */ = {isa = PBXBuildFile; fileRef = AB65933F2A0C52A000DBF059 /* remote_settings.swift */; };
```

Comment:
```suggestion
```



Oh, sorry I didn't catch this earlier—please remove this `remote_settings.swift` line.

(I think the equivalent operation in the Xcode UI is to remove `remote_settings.swift` from the "Compile Sources" section of the "Build Phases" tab of the target?)

There's a rule in Xcode that automatically generates Swift sources from `*.udl` files (in the "Build Rules" tab), and adding the generated `.swift` file as a source actually causes all sorts of problems 😬

Reply:
Thanks for the keen eye! I had hard reset the iOS setup to try and get it working again since I had xcode open when I rebased and things had become kinda funky. Guess I missed a few steps 😅

Reply:
Hmm, it looks like these lines are automatically added when adding these files to the newly created group when following [this documentation](https://github.com/mozilla/application-services/blob/main/docs/howtos/adding-a-new-component.md#the-swift-bindings):
```
Finally, in the Project navigator, add a sub-group named "Generated", pointing to the ./Generated/ subdirectory, and containing entries for the files generated by UniFFI: * <your_crate_name>.swift * <your_crate_name>FFI.h Make sure that "Copy items if needed" is unchecked, and that nothing is checked in "Add to targets".
```

Though I'm also seeing that my local copy does not have a bunch of `.udl` files in the `Compile Sources` section any more either, so I'm wondering if perhaps the documentation has recently become out of date.

Reply:
Oh, how odd! It automatically added them to the target even with nothing checked in "Add to targets"? I wonder if there's another setting that controls that... 🤔 Sorry you had to untangle all that, but it looks like you fixed the build and we're good to go!

I think this is the note about double-checking that the generated `.swift` file doesn't appear in "Compile Sources", though:

> Double-check that `<your_crate_name>.swift` does not appear in the "Compile Sources" section.

(That one bit me in #5346, along with the "Copy Bundle Resources" callout above it that caused a dependency cycle error during the build. Xcode's default behavior unfortunately makes things worse here 😔)

But it's very subtle, and I wonder if we can call it out more explicitly that `<your_crate_name>.swift` is referring to the `Generated/<your_crate_name>.swift`.